src/components/victory-chart/victory-chart.js

```diff
@@ -307,6 +315,7 @@ export default class VictoryChart extends React.Component {
         ref: index,
         key: index,
         standalone: false,
+        theme: props.theme,
```

Comment:
what's the process by which a theme added to victory-chart should be getting passed down to child components?